src/background/index.ts

```diff
@@ -83,6 +83,7 @@ chrome.tabs.onUpdated.addListener(function(tabId, changeInfo) {
 // middleware, can only listen for external messages in background page:
 // https://stackoverflow.com/questions/18835452/chrome-extension-onmessageexternal-undefined
 const authListener = (request) => {
+  console.log("newAuth")
```

Comment:
this logs on every refresh (so authListener must be getting triggered automatically?)